specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -5114,459 +5125,2029 @@
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationServiceEnvironmentName",
+            "description": "The integration service environment name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "integrationServiceEnvironment",
+            "description": "The integration service environment.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationServiceEnvironment"
+            }
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+	    "delete": {
+        "tags": [
+          "IntegrationServiceEnvironments"
+        ],
+        "operationId": "IntegrationServiceEnvironments_Delete",
+        "description": "Deletes an integration service environment.",
+        "x-ms-examples": {
+          "Delete an integration account": {
+            "$ref": "./examples/IntegrationServiceEnvironments_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationServiceEnvironment",
+            "description": "The integration service environment.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Logic error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+	  },
+	  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/integrationServiceEnvironments/{integrationServiceEnvironmentName}/skus": {
```

Comment:
SKUs is a provider level API. It should be /subscriptions/{subscription-id}/providers/Microsoft.Logic/skus. And in this response, the skus for all resource types under the provider must be returned.

Reply:
Please let me know if you need access to the ARM sku contract.

Reply:
As discussed, this API needs to return all Integration Service Environment (resource) Skus for Microsoft Logic RP. So it is modelled at resource level.
The monitoring team want this API for auto-scale of ISE and they wanted it this way so I guess other RPs are also doing something similar.

It won't be possible to change it now without breaking monitoring on ISE so I can remove it from swagger if that's your concern and change it later in consent with Monitoring and ARM team.

Reply:
As discussed waiting for the response you get from Andy's team on this.